client/cityrepdata.cpp

```diff
@@ -726,63 +726,83 @@ static QString cr_entry_cma(const struct city *pcity, const void *data)
 #define FUNC_TAG(var) cr_entry_##var, #var
 
 static const struct city_report_spec base_city_report_specs[] = {
-    {true, -15, 0, nullptr, N_("?city:Name"), N_("City Name"), nullptr,
+    // CITY columns
+    {true, -15, 0, nullptr, N_("?city:Name"), N_("City: Name"), nullptr,
      FUNC_TAG(cityname)},
-    {false, -15, 0, nullptr, N_("Nation"), N_("Nation"), nullptr,
+    {false, -15, 0, nullptr, N_("Nation"), N_("City: Nation"), nullptr,
      FUNC_TAG(nation)},
-    {true, 2, 1, nullptr, N_("?size [short]:Sz"), N_("Size"), nullptr,
+    {false, 3, 1, nullptr, N_("?Continent:C"), N_("City: Continent number"),
+     nullptr, FUNC_TAG(continent)},
+    {true, 2, 1, nullptr, N_("?size [short]:Sz"), N_("City: Size"), nullptr,
      FUNC_TAG(size)},
+    {// TRANS: Header "It will take this many turns before city grows"
+     true, 14, 1, N_("?food (population):Grow"),
+     N_("?Stock/Target:(Have/Need)"), N_("City: Turns until growth/famine"),
+     nullptr, FUNC_TAG(growturns)},
     {true, -8, 1, nullptr, N_("State"),
-     N_("Celebrating/Happy/Peace/Disorder"), nullptr,
+     N_("City: Celebrating/Happy/Peace/Disorder"), nullptr,
```

Comment:
I would change this to "City: Exp State:" to show difference from "City: Concise State:"